docs/make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF
```

Comment:
Can we remove this file? I doubt we use a bat file, or the docs are generated on Windows.